src/main/scala/com/gu/fastly/Lambda.scala

```diff
@@ -23,8 +23,10 @@ class Lambda {
       (event.itemType, event.eventType) match {
         case (ItemType.Content, EventType.Delete) =>
           sendPurgeRequest(event.payloadId)
+        case (ItemType.Content, EventType.Update) =>
+          sendSoftPurgeRequest(event.payloadId)
         case other =>
-          // for now we only send purges for content takedowns, so ignore any other events
+          // for now we only send purges for content, so ignore any other events
```

Comment:
@gklopper Short answer: yes, it's 3. [Here](https://github.com/guardian/content-api-models/blob/master/models/src/main/thrift/events/crier/event.thrift) is the latest version of the file.

Long answer: When I wrote this Lambda we weren't publishing the Thrift file for Crier events to Maven, so I had to copy-paste the Thrift file into this project. But nowadays we are publishing it, as part of the `content-api-models` artifact. I'll make a separate PR to add a dependency on that and delete the copy-pasted file. For now, please just update the file manually.